packages/flutter/lib/src/material/slider.dart

```diff
@@ -698,9 +698,32 @@ class _SliderState extends State<Slider> with TickerProviderStateMixin {
     // in range_slider.dart.
     Size _screenSize() => MediaQuery.of(context).size;
 
+    VoidCallback? handleDidGainAccessibilityFocus;
+    switch (theme.platform) {
+      case TargetPlatform.android:
+      case TargetPlatform.fuchsia:
+      case TargetPlatform.iOS:
+      case TargetPlatform.linux:
+      case TargetPlatform.macOS:
+        break;
+      case TargetPlatform.windows:
+        handleDidGainAccessibilityFocus = () {
+          // Automatically activate the slider when it receives a11y focus.
+          final BuildContext? context = _renderObjectKey.currentContext;
```

Comment:
This is a surprising way to get the focus node, do you know who owns the focus node?

Reply:
Yep, the focus node is owned by the Focus inside the `FocusableActionDetector`, which is why we need to get it from that context.

Reply:
I see it goes very deep into the `FocusableActionDetector`, can make the slider owns the focus node if it is not passed in? One of this internal widget needs to own it, so it doesn't matter which one do

Reply:
When I talked to @gspencergoog about this, his initial thought was we could try making the FocusableActionDetector the parent of the Semantics widget, but that doesn't work, since the end result is creating a 'group' node in the windows tree instead of a slider node. Instead we reach in to grab the focus from the child of the `Focus` object associated with the slider.

Reply:
The current implementation may be fragile, if the internal structure changes, we may not able to find the focus node.

>can we make the slider to own the focus node if it is not passed in? One of this internal widget needs to own it, so it doesn't matter which one do

is this possible?

Reply:
If you wanted, if the `widget.focusNode` wasn't set, you could create your own focusNode (and dispose of it properly) with similar arguments to what the `FocusableActionDetector` does, and pass that in to the detector, and then just talk to either `widget.focusNode` or the one you create (whichever was set)  in your callback.  I do something like this this inside the `Focus` widget State object.

Reply:
Sure, yep. That works fine for me.

Reply:
Done. PTAL.